packages/framework/src/Model/Customer/User/Role/CustomerUserRoleProvider.php

```diff
@@ -42,4 +45,16 @@ public function canCustomerUserSeePrices(?CustomerUser $customerUser): bool
 
         return in_array(CustomerUserRole::ROLE_API_CUSTOMER_SEES_PRICES, $roles, true);
     }
+
+    /**
+     * @return bool
+     */
+    public function canCurrentCustomerUserSeePrices(): bool
+    {
+        if ($this->security->getUser() === null) {
+            return true;
+        }
+
+        return $this->security->getUser() !== null && $this->security->isGranted(CustomerUserRole::ROLE_API_CUSTOMER_SEES_PRICES);
```

Comment:
can be simplified to:
```suggestion
        return $this->security->isGranted(CustomerUserRole::ROLE_API_CUSTOMER_SEES_PRICES);
```

(you have already checked the user is not null in the condition above)

Reply:
fixed: https://github.com/shopsys/shopsys/pull/3355/commits/c3867bb2c06ef9dafd9ead3832440e4a35f5152c